packages/@yoda/flora/src/cli-native.cc

```diff
@@ -490,6 +494,8 @@ static bool genCapsByJSArray(napi_env env, napi_value jsmsg,
       if (!genCapsByJSArray(env, v, sub))
         return false;
       caps->write(sub);
+    } else if (tp == napi_undefined) {
```

Comment:
`null` should also be converted to caps void.

Reply:
flora msg not support push 'object'